template_sets.go

```diff
@@ -187,18 +187,20 @@ func (set *TemplateSet) BanTag(name string) {
 
 // Ban a specific filter for this template set. See more in the documentation for TemplateSet.
 func (set *TemplateSet) BanFilter(name string) {
-	_, has := filters[name]
-	if !has {
-		panic(fmt.Sprintf("Filter '%s' not found.", name))
-	}
-	if set.firstTemplateCreated {
-		panic("You cannot ban any filters after you've added your first template to your template set.")
-	}
-	_, has = set.bannedFilters[name]
-	if has {
-		panic(fmt.Sprintf("Filter '%s' is already banned.", name))
-	}
-	set.bannedFilters[name] = true
+	// TODO: implement this, but so far unused
```

Comment:
I left it while I was changing code to come back later to it. However we don't really need this because we control our usage of filters.